### ISSUE TITLE
How do I run Linux?

### ISSUE THREAD

**Issue by kenorb** [asmblah/jemul8#14]
How do I run Linux on this JS Emulator similar to [JSLinux](http://bellard.org/jslinux/) ([GH](https://github.com/levskaya/jslinux-deobfuscated))?

**Comment by orklann** [comment]
Afaik, jemul8 can not run Linux, because at least it does not support memory paging.
May need the author to declare this issue officially.

**Comment by asmblah** [comment]
That's correct, Linux is not (yet) supported due to the lack of paging support as mentioned @aaron-elkins, amongst other missing components. There will be some updates soon, though. :smile:
